.github/workflows/quality-api.yml

```
@@ -167,10 +167,11 @@ jobs:
-        working-directory: api-e2e
+        working-directory: api
         env:
-          APP_POSTGRES_URL: postgres://postgres:postgres@localhost:5432/test
-          APIE2E_API_URL: http://api.covoiturage.test
-          APIE2E_AUTH_USERNAME: admin@example.com
-          APIE2E_AUTH_PASSWORD: password
-        run: just ci
+          POSTGRES_DB: test
+          APP_POSTGRES_URL: postgres://postgres:postgres@postgres:5432/test
+        run: just ci_test_e2e
+        resources:
```

actionlint (1.7.4)

unexpected key "resources" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)